src/gcp_scanner/crawl.py

```diff
@@ -434,8 +414,7 @@ def get_gke_clusters(
   parent = f"projects/{project_name}/locations/-"
   try:
     clusters = gke_client.list_clusters(parent=parent)
-    return [(cluster.name, cluster.description) for cluster in clusters.clusters
-           ]
+    return list(map(lambda x: (x.name,x.description),clusters.clusters))
```

Comment:
Why did you change this from the list comprehension to this? This is less readable.

Reply:
reverted the map() to list comprehension